Catappult/appcoins-sdk
# [fix] Fallback payflow when service installed

**What does this PR do?**

   The empty payflow response was being handled the same way a failed request, but the intention is prevent binding the service if the service response is empty.
   Creates workaround for the install flow to call the wallet since the this flow is specific for the wallet for now.

**Database changed?**

   No

**How should this be manually tested?**

 - Open the SDK with no service installed
 - Install the wallet with the install flow
 - Open the SDK with the wallet installed 
 - Install the GamesHub 
 - Open the SDK with the wallet and GamesHub installed, and check the priorities
 - Uninstall the Wallet
 - Open the SDK with the GamesHub installed 

**Code Review Checklist**
